test/spec/modules/stroeerCoreBidAdapter_spec.js

```diff
@@ -1021,6 +1052,14 @@ describe('stroeerCore bid adapter', function() {
           assert.notProperty(serverRequestInfo, 'uids');
         });
 
+        it('should be able to build without userIdAsEids', () => {
+          const bidReq = buildBidderRequest();
+          bidReq.bids.forEach(bid => delete bid.userIdAsEids);
+          const serverRequestInfo = spec.buildRequests(bidReq.bids, bidReq)[0];
+          assert.lengthOf(serverRequestInfo.data.bids, 2);
+          assert.notProperty(serverRequestInfo, 'uids');
```

Comment:
Oppsie. Copy and paste gone wrong. 'uids' should be 'eids'

And you highlighted two oppsies from me as well. The test you copied from is also wrong. It's not `uids`, its `euids`. Also, we are not asserting the field on the correct level within serverRequestInfo.

Should be something like this:
```javascript
assert.notProperty(serverRequestInfo.data.user, 'eids'); // and 'euids' for the other test
```

Not sure if that will work. If not try:

```javascript
assert.isUndefined(serverRequestInfo.data.user.eids); // and .euids for the other test
```

Not sure what happened there 😅 . Bit risky with these type of assertions. Might want to make sure you see it fail by temporarily commenting out `bidReq.bids.forEach(bid => delete bid.userIdAsEids);`

Reply:
Sorry I have over looked this. Update both tests. Made sure the test fails when comment out the implementation and passes with implementation.